forms/ConfirmedPasswordField.php

```diff
@@ -166,6 +166,9 @@ public function Field($properties = array()) {
 			/** @var FormField $field */
 			$field->setDisabled($this->isDisabled());
 			$field->setReadonly($this->isReadonly());
+			if(!$this->canBeEmpty){
+				$field->setAttribute('required', true);
```

Comment:
because this happens at rendertime it means the dev has no way of turning this off if they want to...

ie: it breaks the usecase of iterating over fields to disable the `required` attribute...

Can we check if it has or hasn't already been set?